.php_cs.dist

```diff
@@ -6,6 +6,7 @@ $config = PhpCsFixer\Config::create()
         '@Symfony' => true,
         'array_syntax' => ['syntax' => 'short'],
         'single_line_throw' => false,
+        'protected_to_private' => false,
```

Comment:
Adding this setting to ignore final class with private properties checking.

Reply:
is this about some other class? because Wsse has no protected properties/methods.

Reply:
This setting is for following coding style checking issue:

```
Loaded config default from "/github/workspace/.php_cs.dist".
   1) src/CookieJar.php
      ---------- begin diff ----------
--- Original
+++ New
@@ -12,7 +12,7 @@
     /**
      * @var \SplObjectStorage
      */
-    protected $cookies;
+    private $cookies;
 
     public function __construct()
     {
@@ -92,7 +92,7 @@
      *
      * @return Cookie[]
      */
-    protected function findMatchingCookies(callable $match)
+    private function findMatchingCookies(callable $match)
     {
         $cookies = [];
 

      ----------- end diff -----------

   2) src/Encoding/FilteredStream.php
      ---------- begin diff ----------
--- Original
+++ New
@@ -13,12 +13,11 @@
  */
 abstract class FilteredStream implements StreamInterface
 {
-    const BUFFER_SIZE = 8192;
-
     use StreamDecorator {
         rewind as private doRewind;
         seek as private doSeek;
     }
+    const BUFFER_SIZE = 8192;
 
     /**
      * @var callable

      ----------- end diff -----------

```

Reply:
ah i see. i fixed those method visibilities in the master branch, we can remove the config change.
```suggestion
```

Reply:
Thanks. This setting has been removed.